Enrich_methods/example.py

```diff
@@ -0,0 +1,210 @@
+    Li2TiO3Li60 = Material()
+    Li2TiO3Li60.from_atom_frac(
+        {enriched_lithium(enrichment): 4, 80000000: 4, 140000000: 1}
+    )
+    Li2TiO3Li60.density = 2.40
+    Li2TiO3Li60 = Li2TiO3Li60.expand_elements()
+    Li2TiO3Li60.metadata["citation"] = "HernandezFusEngDes_2018"
+    return Li2TiO3Li60
+
+
+# I also need to have another version of this function for the pin breeder zone.
+# I will if need to either make a version of this for each material I'm testing
+# or I will need a universial function that will require me to pass all the arugments in hcpb_pin_bw()
+# plus whatever material I'm testing for the case. Currently I'm testing around 8 different materials
+# How I make the universal function will depend on how I can import the library or if I can use enrich
+def hcpb_pin_bw(
+    enrichment=0.6,
+    Li4SiO4_frac=0.065,
+    Li2TiO3_frac=0.035,
+    he_frac=0.482,
+    EUROFER97_frac=0.418,
+):
+
+    Li4SiO4Li = Li4SiO4Li_mat(enrichment)
+    Li2TiO3Li = Li2TiO3Li_mat(enrichment)
+
+    mix = MultiMaterial(
+        {
+            mat_lib_test["EUROFER97"]: EUROFER97_frac,
+            Li4SiO4Li: Li4SiO4_frac,
+            mat_lib_test["HeT410P80"]: he_frac,
+            Li2TiO3Li: Li2TiO3_frac,
+        }
+    )
+    hcpb_pin_bw = mix.mix_by_volume()
+    hcpb_pin_bw.metadata["constituentcitation"] = "HernandezFusEngDes_2018"
+    return hcpb_pin_bw
+
+
+################################### Method 2
+
+# Method if you want to use enrich function without it being called in mix_by_volume or in make_mat_from_atom
+# But this version uses the output json file which will give you the same answer as rebuiling the material to the second last decimal
+# Earlier you said that you wanted to avoid that method since you don't trust it
+enriched_mat = mdbt.Material({30060000: 0.6, 30070000: 0.4})
+collapsed = mat_lib_test["Li2TiO3nat"].collapse_elements({3})
+atom_frac_enriched = mdbt.enrich(
+    collapsed.to_atom_frac(), 30000000, enriched_mat.to_atom_frac()
+)
+
+mat_input2 = {
+    "atom_frac": atom_frac_enriched,
+    "density": 3.42,
+    "citation": "HernandezFusEngDes_2018",
+}
+material2 = mdbt.make_mat_from_atom(**mat_input2)
+# From here you can either add it to the input library or call it in def hcpb_pin_bw
+# Benefit of adding to input library you can have it included in your output material library like ss 316 currently
+
+
+################################### Method 3
+
+# You can either use "from material_library import mat_data as pure" or just redefine the material which would be the same as the current method
+
+enriched_mat = mdbt.Material({30060000: 0.6, 30070000: 0.4})
+atom_frac_enriched = mdbt.enrich(
+    pure["Li2TiO3nat"]["atom_frac"], 30000000, enriched_mat.to_atom_frac()
+)
+# I would need to call enrich() each time I wanted to enrich a different element
+mat_input3 = {
+    "atom_frac": atom_frac_enriched,
+    "density": 3.42,
+    "citation": "HernandezFusEngDes_2018",
+}
```

Comment:
This is definitely better without the need to collapse elements, and could be streamlined with a single line to get the enriched atom fractions.
```suggestion
enriched_atom_frac = mdbt.Material({30060000: 0.6, 30070000: 0.4}).to_atom_frac()

# I would need to call enrich() each time I wanted to enrich a different element
mat_input3 = {
    "atom_frac": mdbt.enrich(pure["Li2TiO3nat"]["atom_frac"], "Li" , enriched_atom_frac)
    "density": 3.42,
    "citation": "HernandezFusEngDes_2018",
}
```

Reply:
This might suggest that it makes sense to have an `enrich()` method that takes a `mat_data` entry instead of a material composition???